backend/profiles/models.py

```diff
@@ -1,3 +1,27 @@
 from django.db import models
-
+from django.contrib.auth.models import User
 # Create your models here.
+
+class Startup(models.Model):
+    user = models.ForeignKey(User, on_delete=models.CASCADE, related_name='startups')
+    company_name = models.CharField(max_length=150)
+    description = models.TextField(blank=True)
+    website = models.URLField(blank=True)
+    views_count = models.IntegerField(default=0)
```

Comment:
This field can be left, although it is not needed now, but it is good that the default is "0".